fabfile.py

```diff
@@ -20,6 +18,55 @@
 # Tasks
 # ---------------------------------------------------------------------------------------
 
+# Launch an AWS instance and install testy using the given WiredTiger and testy branches.
+@task
+def launch(c, distro, wiredtiger_branch="develop", testy_branch="main"):
+
+    result = launch_from_distro(distro)
+    if result['status'] != 0:
+        print(f"Launch failed. {result['msg']}")
+        return
+
+    user = result['user']
+    hostname = result['hostname']
+
+    try:
```

Comment:
I noticed the previous launch function had a number of retries here when creating a installing wiredtiger and testy. Is failing to establish a connection or install these on the first attempt likely to happen?

Reply:
The previous version was using retries to wait on the various bits and pieces to be ready for use (such as the storage volume and the networking). AWS has status checks that we can use to serve the same purpose, so there's no need to do both, and the AWS status check is the correct way to determine that the instance is 100% ready for use.

Reply:
Ah okay I see so the status check above is sufficient. Thanks!